src/components/scrollbar-container/index.tsx

```diff
@@ -34,6 +34,11 @@ export class ScrollbarContainer extends React.Component<Properties, State> {
   }
 
   componentDidUpdate(prevProps: Properties) {
+    if (prevProps.children !== this.props.children) {
+      const div = document.getElementById('scrollbar-container__content');
```

Comment:
We should use a ref here rather than fetching by id.

Reply:
Haha, actually, there's already a ref for that node.... we should be able to use `this.scrollContainerRef.current`

Reply:
In this case they both happened to work but they are not the same at all.

Using the document `getElementById` queries the whole DOM and will get all elements that have id. If you have more than one scroll container then you don't know which one you'll get. Using `ref`s is a React mechanism for getting the actual rendered DOM element associated with the component. It should be used very rarely because you're supposed to manage a component in "react-y" ways. But for certain things you do sometimes need access. This is one of them.

Reply:
got it.